src/wallet/test/psbt_wallet_tests.cpp

```diff
@@ -35,15 +35,15 @@ BOOST_AUTO_TEST_CASE(psbt_updater_test)
     s_rs1 >> rs1;
     spk_man->AddCScript(rs1);
 
-    CScript rs2;
+    /*CScript rs2;
     CDataStream s_rs2(ParseHex("2200208c2353173743b595dfb4a07b72ba8e42e3797da74e87fe7d9d7497e3b2028903"), SER_NETWORK, PROTOCOL_VERSION);
     s_rs2 >> rs2;
-    spk_man->AddCScript(rs2);
+    spk_man->AddCScript(rs2);*/
 
-    CScript ws1;
+    /*CScript ws1;
     CDataStream s_ws1(ParseHex("47522103089dc10c7ac6db54f91329af617333db388cead0c231f723379d1b99030b02dc21023add904f3d6dcf59ddb906b0dee23529b7ffb9ed50e5e86151926860221f0e7352ae"), SER_NETWORK, PROTOCOL_VERSION);
     s_ws1 >> ws1;
-    spk_man->AddCScript(ws1);
+    spk_man->AddCScript(ws1);*/
```

Comment:
This doesn't need to block the merge, but I'm curious if we can delete this code altogether now, rather than leave it commented out.

Reply:
Good question. I am not too worried about leaving it here since we need to rewrite this test to not use segwit. (I created an item in the big plan under the "disable segwit" section where this test is pointed out.)

I.e. we have to edit this test out to be sure that PSBT works without segwit activation, so this entire file will be reworked.